apps/artwork/components/related_artworks/query.coffee

```diff
@@ -12,5 +12,8 @@ module.exports = """
         ... artwork_brick
       }
     }
+    related(size: 1) {
```

Comment:
Is this just here to check if there are any related works (in order to know whether to render the tab)?

if so, maybe there should be a counts thing in metaphysics for this? not sure what the best practice is for that. #minortho